packages/components/Tabs/src/Tabs.windows.tsx

```diff
@@ -0,0 +1,187 @@
+/** @jsx withSlots */
+import * as React from 'react';
+import { View } from 'react-native';
+import { Text } from '@fluentui-react-native/text';
+import { tabsName, TabsType, TabsProps, TabsState, TabsSlotProps, TabsRenderData, TabsContextData } from './Tabs.types';
+import { compose, IUseComposeStyling } from '@uifabricshared/foundation-compose';
+import { ISlots, withSlots } from '@uifabricshared/foundation-composable';
+import { settings } from './Tabs.settings';
+import { mergeSettings } from '@uifabricshared/foundation-settings';
+import { filterViewProps } from '@fluentui-react-native/adapters';
+import { foregroundColorTokens, textTokens, backgroundColorTokens } from '@fluentui-react-native/tokens';
+import { useSelectedKey, useAsPressable } from '@fluentui-react-native/interactive-hooks';
+
+export const TabsContext = React.createContext<TabsContextData>({
+  selectedKey: null,
+  onTabsClick: (/* key: string */) => {
+    return;
+  },
+  getTabId: (/* key:string, index: number*/) => {
+    return null
+  },
+  updateSelectedTabsItemRef: (/* ref: React.RefObject<any>*/) => {
+    return;
+  },
+  tabsItemKeys: [],
+  views: null,
+  focusZoneRef: null,
+});
+
+export const Tabs = compose<TabsType>({
+  displayName: tabsName,
+
+  usePrepareProps: (userProps: TabsProps, useStyling: IUseComposeStyling<TabsType>) => {
+    const focusZoneRef = React.useRef(null);
+    const defaultComponentRef = React.useRef(null);
+    const {
+      label,
+      accessibilityLabel = userProps.label,
+      selectedKey,
+      headersOnly,
+      defaultSelectedKey,
+      getTabId,
+      componentRef = defaultComponentRef,
+      isCircularNavigation,
+      ...rest
+    } = userProps;
+
+    // This hook updates the Selected TabsItem and calls the customer's onTabsClick function. This gets called after a TabsItem is pressed.
+    const data = useSelectedKey(selectedKey || defaultSelectedKey || null, userProps.onTabsClick);
+
+    const [, setSelectedTabsItemRef] = React.useState(React.useRef<View>(null));
```

Comment:
`[, setSelectedTabsItemRef]` Was this a mistake? Or is this some syntax I've never seen before?

Reply:
Related, I think it would look cleaner if the useRef hook wasn't inline like that. Especially for hooks, I think it's important to keep them top level and not nested to be sure they aren't called conditionally and such.

Reply:
It was on purpose, it omits the first element and just assigns the second one. This is because selectedTabsItemRef is not needed for the windows implementation.

Reply:
What do you mean by the second one? If `selectedTabsItemRef ` is not needed for the Windows implementation, does this provide any value?

Reply:
It does not, will be removing the hook and references to it.